src/graphql/data/fetchGraphQL.ts

```diff
@@ -16,7 +16,12 @@ const fetchQuery = (params: RequestParameters, variables: Variables): Promise<Gr
     variables,
   })
 
-  return fetch(URL, { method: 'POST', body, headers }).then((res) => res.json())
+  return fetch(URL, { method: 'POST', body, headers })
+    .then((res) => res.json())
+    .catch((e) => {
+      console.error(e)
+      return { data: [] }
+    })
```

Comment:
Also, I add a `catch` statement to prevent the app crashed:
<img width="1439" alt="Screen Shot 2022-10-14 at 08 39 40" src="https://user-images.githubusercontent.com/31362988/195736778-933fe6fc-5181-45b9-adb9-910ef20ca413.png">

I also suggest that the team can [set up Sentry support ](https://github.com/Uniswap/interface/issues/3611) to debug all of the errors in the production environment.

Reply:
This should return { error: ErrorObject }, but it can be fixed in a follow-up. This should do a good job of fixing the crash for now.